fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/data/LoanAccountData.java

```diff
@@ -239,6 +240,7 @@ public final class LoanAccountData {
     private LocalDate expectedDisbursementDate;
 
     private final CollectionData delinquent;
```

Comment:
@josehernandezfintecheandomx What is the deal with this "other delinquent" logic?

Reply:
It was to store some attributes related to delinquency too, please review the next ticket https://mifosforge.jira.com/browse/PS-600